### ISSUE TITLE
React LGV can't load ESM plugins

### ISSUE THREAD

**Issue by garrettjstevens** [GMOD/jbrowse-components#2638]
If you're using `@jbrowse/react-linear-genome-view` and are loading external plugins, this works:

```js
const loadedPlugins = await loadPlugins([
  {
    name: 'LogRegion',
    url: 'http://localhost:9000/dist/jbrowse-plugin-log-region.umd.development.js',
  },
])
```

but this does not:

```js
const loadedPlugins = await loadPlugins([
  {
    esmUrl: 'http://localhost:9000/dist/jbrowse-plugin-log-region.esm.js',
  },
])
```

The error message isn't too informative, it says `Uncaught (in promise) Error: Cannot find module 'http://localhost:9000/dist/jbrowse-plugin-log-region.esm.js'`

I think the dynamic `import` in the plugin loader that is used for loading ESM is getting transpiled to a `requre` by babel, which doesn't work for this use case. Maybe we could make an ESM build for core?

**Comment by cmdcolin** [comment]
i'd definitely be interested in seeing less babel-ification...recently the parser packages made it so they have an esm build using just tsc! worked pretty well i'd say

**Comment by cmdcolin** [comment]
fixed via #2891 
could use an example of how other bundlers can do it. bundlers really don't like dynamic / "expression" imports (they like to know it statically ahead of time...) so we use webpackIgnore but dunno what other bundlers use
